packages/renderer/src/lib/image/ImageDetailsFiles.svelte

```diff
@@ -65,9 +66,6 @@ onDestroy(() => {
   </div>
 {:else if imageLayers}
   <div class="flex flex-col w-full h-full p-8 pr-0 text-[var(--pd-content-text)] bg-[var(--pd-content-bg)]">
-    <div class="pr-4">
-      <slot name="header-info" />
-    </div>
```

Comment:
how is it replaced now ?

Reply:
I removed this part as it is never used: this component is inspired from the `ProviderResultPage` component, where this slot is used, but is never used in `ImageDetailsFiles`. 

Also, moving to svelte5, the checks refuses to use this svelte4 notation, I had to change or remove it.